troposphere/static/js/public_site/bootstrapper.jsx

```diff
@@ -16,12 +16,17 @@ browserBondo.conditionalFill();
 // Register which stores the application should use
 import stores from "stores";
 
+stores.ProviderStore = require("stores/ProviderStore");
```

Comment:
PasswordLoginForm.jsx used this store from the "context" of the public_site, that's why this is added to bootstrapper.